src/EditorFeatures/Core/Shared/SuggestionSupport/IDocumentSupportsSuggestionService.cs

```diff
@@ -9,5 +9,6 @@ internal interface IDocumentSupportsSuggestionService : IWorkspaceService
         bool SupportsCodeFixes(Document document);
         bool SupportsRefactorings(Document document);
         bool SupportsRename(Document document);
+        bool SupportsGoToNextPreviousMethod(Document document);
```

Comment:
If we're trying to surface the concept of "Trust us, you're not stuck in a *weird* projection", then maybe "SupportsPositionalNavigation" or "SupportsNavigationToAnyPosition" or "SupportsViewingAnyPosition" or .......